Files/Views/MainPage.xaml.cs

```diff
@@ -186,7 +186,7 @@ private async void SidebarControl_SidebarItemInvoked(object sender, SidebarItemI
             if (invokedItemContainer.DataContext is MainPageViewModel)
             {
                 SettingsDialog settingsDialog = new SettingsDialog();
-                await settingsDialog.ShowAsync();
+                _ = await settingsDialog.ShowAsync();
```

Comment:
what's the difference here 🤔

Reply:
None I suppose, it just fixes [this warning](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0058), and makes the code cleaner.